### ISSUE TITLE
Visualization

### ISSUE THREAD

**Issue by zhuzhen1996** [polimi-ispl/icpr2020dfdc#44]
Hello, don't bother to disturb you again! I trained several models according to the division of my data set. But I want to check the training process and what the loss is. I think you are using tensorboardX, how do you visualize the training process?
![image](https://user-images.githubusercontent.com/55490753/115944046-27acc700-a4e6-11eb-82d4-40fc16d04303.png)

**Comment by nicobonne** [comment]
Hi, it's as simple as running tensorboard in the `--log_dir` folder you provided to the training script.
```bash
cd $YOURLOGDIR
tensorboard --logdir .
```
then tensorboard will initialize a web server on a certain port on localhost, open your browser and go to `localhost:port`

**Comment by zhuzhen1996** [comment]
I have understanding it now,Thank you very much!



发自我的iPhone


------------------ Original ------------------
From: Nicolò Bonettini ***@***.***&gt;
Date: Sat,Apr 24,2021 8:11 PM
To: polimi-ispl/icpr2020dfdc ***@***.***&gt;
Cc: zhuzhen1996 ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [polimi-ispl/icpr2020dfdc] Visualization (#44)





 
Hi, it's as simple as running tensorboard in the --log_dir folder you provided to the training script.
 cd $YOURLOGDIR tensorboard --logdir .
 
then tensorboard will initialize a web server on a certain port on localhost, open your browser and go to localhost:port
